app/src/main/java/io/github/dotslash21/faclient/AuthActivity.java

```diff
@@ -20,7 +20,6 @@
 import android.content.pm.ActivityInfo;
 import android.content.pm.PackageInfo;
 import android.content.pm.PackageManager;
-import android.hardware.Camera;
```

Comment:
Why are you using android.hardware.Camera.* ??

Reply:
obsolete dekhachilo kichu tai auto resolve hoeche.jekhane camera.get camera number korchilis okhane just get camera number die e kaj hoe jabe..nothing else